plantcv/plantcv/visualize/obj_size_ecdf.py

```diff
@@ -0,0 +1,47 @@
+# Plot Empirical Cumulative Distribution Function for Object Size
+
+import os
+import cv2
+import pandas as pd
+import numpy as np
+from plantcv.plantcv import params, fatal_error
+from plantcv.plantcv._debug import _debug
+from statsmodels.distributions.empirical_distribution import ECDF
+from plotnine import ggplot, aes, geom_point, labels, \
+    scale_color_manual, scale_x_log10
```

Comment:
```suggestion
from plotnine import ggplot, aes, geom_point, labels, scale_x_log10
```

Remove unused import